libs/service-portal/access-control/delegations/src/screens/AccessControl/AccessControl.tsx

```diff
@@ -0,0 +1,152 @@
+  useNamespaces('sp.settings-access-control')
+  const { data, loading } = useQuery<Query>(AuthDelegationsQuery)
+  const { switchUser } = useAuth()
+  const { formatMessage } = useLocale()
+  const history = useHistory()
+  const location = useLocation()
+  const isDelegationToMe =
+    location.pathname === ServicePortalPath.AccessControlDelegationsToMe
+
+  if (!loading && data?.authDelegations.length === 0) {
+    return (
+      <NoDataScreen
+        title={formatMessage({
+          id: 'sp.settings-access-control:empty-title',
+          defaultMessage: 'Umboð',
+        })}
+        button={{
+          type: 'internal',
+          link: ServicePortalPath.AccessControlDelegationsGrant,
+          text: formatMessage({
+            id: 'sp.settings-access-control:empty-new-access',
+            defaultMessage: 'Veita aðgang',
+          }),
+          variant: 'primary',
+        }}
+        secondaryButton={{
+          type: 'click',
+          onClick: () => switchUser(),
+          text: formatMessage({
+            id: 'sp.settings-access-control:empty-switch-access',
+            defaultMessage: 'Skipta um notanda',
+          }),
+          variant: 'ghost',
+        }}
+      >
+        <Text>
+          {formatMessage({
+            id: 'sp.settings-access-control:empty-intro',
+            defaultMessage:
+              'Hérna kemur listi yfir þau umboð sem þú hefur gefið öðrum. Þú getur eytt umboðum eða bætt við nýjum.',
+          })}
+        </Text>
+      </NoDataScreen>
+    )
+  }
+  const actor = userInfo.profile.actor
+  const isDelegation = Boolean(actor)
+
+  const isCompany = userInfo.profile.subjectType === 'legalEntity'
+  const personDelegation = isDelegation && !isCompany
+
+  if (personDelegation) {
+    return <AccessDenied userInfo={userInfo} client={client} />
+  }
+
+  const creatTab = (
+    label: MessageDescriptor,
+    headerTitle: MessageDescriptor,
+  ) => ({
+    label: formatMessage(label),
+    content: (
+      <Box marginTop={8}>
+        <IntroHeader
+          title={headerTitle}
+          intro={formatMessage({
+            id: 'sp.access-control-delegations:header-intro',
+            defaultMessage:
+              'Hérna kemur listi yfir þau umboð sem þú hefur gefið öðrum. Þú getur eytt umboðum eða bætt við nýjum.',
+          })}
+        />
+        <Accesses />
+      </Box>
+    ),
+  })
+
+  const delegationFromMeTabId = '0'
+  const delegationToMeTabId = '1'
+
+  const tabClickHandler = (id: string) => {
+    const url =
+      id === delegationToMeTabId
+        ? ServicePortalPath.AccessControlDelegationsToMe
+        : ServicePortalPath.AccessControlDelegations
+
+    // Make sure not to add to history stack the same route twice in a row
+    if (url !== location.pathname) {
+      history.push(url)
+    }
+  }
+
+  return (
+    <Box>
+      {!loading && data && data?.authDelegations?.length > 0 && (
+        <Tabs
```

Comment:
Should skip the tabs for now since they come way later in M2?

Reply:
I have updated the code a bit and the content for to me delegation tab is now null. I think it won't be long until @saevarma has updated the query to support direction outgoing and incoming.
Let me know if you still want me to remove until then